forest/cli/src/cli/wallet_cmd.rs

```diff
@@ -23,9 +23,21 @@ use fvm_shared::{
 };
 use num::BigInt;
 use rpassword::read_password;
+use rust_decimal::prelude::*;
+use rust_decimal_macros::dec;
 
 use super::{handle_rpc_err, Config};
 
+const NUM_SIGNIFICANT_DIGITS: u32 = 4;
+
+#[allow(clippy::enum_variant_names)]
+enum FormattingMode {
```

Comment:
Some short and concise documentation for the formatting modes would be great.

Reply:
Added